File: uri/1014.go
package main

import (
	"fmt"
)

func main() {
	var rendimento, gasto float32
	var distancia int
	fmt.Scanf("%d\n%f", &distancia, &gasto)
	rendimento = float32(distancia) / gasto
	fmt.Printf("%.3f km/l\n", rendimento)
}

File: EXERCICIOS/uri/1014.c
#include<stdio.h>

int main () {
    float rendimento, gasto;
    int distancia;
    scanf("%i %f", &distancia, &gasto);
    rendimento = (float)distancia / gasto;
    printf("%.3f km/l\n", rendimento);
    return 0;
}
